.github/workflows/build.yaml

```diff
@@ -23,6 +23,6 @@ jobs:
           go-version: 1.16
       - name: Build & Test
         run: |
-          go build -v ./cmd/controller
+          go build -v ./cmd/operator
```

Comment:
I'd keep this as controller. It's a bit pedantic, but I consider the entire software installation (a set of controllers and webhooks) to together be an operator. The binary itself is the controller, which may be the core of the operator, but isn't itself the operator.

Reply:
>but I consider the entire software installation (a set of controllers and webhooks) to together be an operator
Isn't this a complete installation (with set of controllers and webhooks), like whatever we need to create clusters will part of this binary?
The confusing part is, we have this project named as kit/operator and its binary is named controller?

Reply:
We will likely have two binaries: controller and webhook. Perhaps even more binaries for different things. These are all pieces of the "operator"

Reply:
e.g. Karpenter is a kubernetes operator with a controller and a webhook.